manage_sections/views.py

```diff
@@ -122,6 +119,7 @@ def _add_badge_label_name_to_enrollments(enrollments):
 @lti_permission_required('manage_sections')
 @require_http_methods(['GET'])
 def create_section_form(request):
+    print('IN CREATE SECTION!!')
```

Comment:
Maybe change this to a logger or remove it?

Reply:
There are a few more `print()` statements below that should also be removed or converted to loggers.

Reply:
cleaned up the prints